.circleci/config.yml

```diff
@@ -169,7 +169,6 @@ gn-build-steps: &gn-build-steps
           gclient config \
             --name "src/electron" \
             --unmanaged \
-            --cache-dir "${GIT_CACHE_PATH}" \
```

Comment:
This cache dir is still needed.

Reply:
It's picked up by the environment variable; the command line arg is redundant